opds_springer/book_saver.py

```diff
@@ -40,23 +40,37 @@ def save_books(self):
                         series_id=kbart_row["parent_publication_title_id"],
                         language=springer_data["language"],
                         description=springer_data["description"],
+                        published=springer_data["publication_date"],
                     )
-                    session.add(book)
+                    if springer_data.get("authors"):
```

Comment:
I notice that book_saver always has a list for `links`, but returns None for the contributor fields if there are no values. Would it be disruptive to handle these list fields the same way, and remove the conditional assignment here?

Reply:
So that an empty string is saved? Looking at the feed_generator, I don't _think_ that the behavior of the feed generator would need to change.